### PR TITLE
Order add order model

### PR DESCRIPTION
**Pull request by JoshEvan**

### CODE REVIEW

File: src/main/java/com/compfest/sea/entity/user/model/User.java [hunk 1]
```diff
@@ -57,7 +64,13 @@
   private Merchant merchant;
 
   public User(
-      int id, String name, String email, String password, String phone, String address, Role role) {
+      int id,
+      String najdme,
```
Comment (alvinfebriando) [inline src/main/java/com/compfest/sea/entity/user/model/User.java]: Typo name => najdme

Comment (JillyCS15) [inline src/main/java/com/compfest/sea/entity/user/model/User.java]: ini typo yaa

Comment (JoshEvan) [reply src/main/java/com/compfest/sea/entity/user/model/User.java]: ohhhhh wkwkw, thank you, sorry

**Review by alvinfebriando — APPROVED** [review]

File: src/main/java/com/compfest/sea/entity/user/model/User.java (hunk 1 above)
Comment (JoshEvan) [reply src/main/java/com/compfest/sea/entity/user/model/User.java]: udah ya barusan, sorryyy

**Review by JillyCS15 — APPROVED** [review]

**Pull request merged**